docker/bash-scripts/ecr-script/docker-ecr.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+DockerRepository="";
+ProjectName="";
+Region="";
+
```

Comment:
I would add a check to make sure these values aren't empty with some instructions that they need to modify the file and populate them.